Resources/doc/reference/architecture.rst

```diff
@@ -354,6 +355,41 @@ Then, you have to set the CommentAdmin ``parentAssociationMapping`` attribute to
         }
     }
 
+To display the ``CommentAdmin`` extend the menu in your ``PostAdmin`` class:
+
+.. code-block:: php
+
+    <?php
+
+    namespace AppBundle\Admin;
+
+    class PostAdmin extends Admin
+    {
+        // ...
+
+        protected function configureTabMenu(MenuItemInterface $menu, $action, AdminInterface $childAdmin = null)
+        {
+            if (!$childAdmin && !in_array($action, array('edit', 'show'))) {
+
+                return;
+            }
+
+            $admin = $this->isChild() ? $this->getParent() : $this;
+            $id = $admin->getRequest()->get('id');
+
+            $menu->addChild('Show Post', array('uri' => $admin->generateUrl('show', array('id' => $id))));
+            if (is_granted('EDIT') {
+                $menu->addChild('Edit Post', array('uri' => $admin->generateUrl('edit', array('id' => $id))));
+            }
+
+            if (is_granted('LIST', $admin) {
+                $menu->addChild('Manage Comments', array(
+                    'uri' => $admin->generateUrl('sonata.news.admin.comment.list', array('id' => $id))
+                ));
+            }
+        }
+    }
+
 It also possible to set a dot-separated value, like ``post.author``, if your parent and child admins are not directly related.
```

Comment:
I would like to add more details to follow up this comment as I don't understand it myself. If someone will explain it here, I will gladly expand on it in the documentation.